### PR TITLE
Improve contributing_documention guide

### PR DESCRIPTION
**Pull request by bthng**
Current guide is specific to Windows OS. 

Issue #805

### CODE REVIEW

File: docs/contributor_guide/contributing_documentation.rst [hunk 1]
```diff
@@ -49,6 +49,14 @@ the following commands:
     ...
     start docs/_build/html/auto_examples/plot_*.html
 
+The above code block works for WindowsOS.
+For MacOS users, use :code: `open docs/_build/html/index.html`
+For Linux users, use :code: `xdg-open docs/_build/html/index.html`
+
+.. note::
+
+    The rendered html files can be explored in any file explorer (e.g. chrome/safari)
```
Comment (romanlutz) [inline docs/contributor_guide/contributing_documentation.rst:58]:
```suggestion
    The rendered HTML files can be explored in any file explorer and then opened using a browser (e.g., Chrome/Safari/Firefox).
```

File: docs/contributor_guide/contributing_documentation.rst [hunk 2]
```diff
@@ -49,6 +49,14 @@ the following commands:
     ...
     start docs/_build/html/auto_examples/plot_*.html
 
+The above code block works for WindowsOS.
```
Comment (romanlutz) [inline docs/contributor_guide/contributing_documentation.rst:52]:
```suggestion
The above code block works for Windows users.
```

**Review by romanlutz — COMMENTED** [review]
Two small suggestions. You can accept them directly in the PR or make adjustments yourself and push new commits.

Thanks for your contribution @bthng !!!

File: docs/contributor_guide/contributing_documentation.rst (hunk 2 above)
Comment (bthng) [reply docs/contributor_guide/contributing_documentation.rst]: I have addressed the following feedback, thank you!

**Review by romanlutz — APPROVED** [review]
Thanks for extending this to be reflective of other users' experiences!

**Review by hildeweerts — APPROVED** [review]

**Pull request merged**